sdk/canton/BUILD.bazel

```diff
@@ -1458,7 +1457,12 @@ da_java_library(
         ":ledger-api-java",
         ":ledger-api-java-grpc",
     ],
-    javacopts = da_java_bindings_javacopts,
+    javacopts = [
```

Comment:
Would it make sense to define that as `da_java_bindings_javacopts + ["-Xlint:-deprecation"]`, so that if `da_java_bindings_javacopts` changes this changes too?